source/getting-started/javascript.md

```diff
@@ -90,48 +95,269 @@ See also: [app authorization in the API reference](/reference/#authorizing-your-
 var connection = new pryv.Connection(credentials);
 ```
 
+### Request access informations
 
-### Retrieve & manipulate events
+```javascript
+connection.accessInfo(function (err, accessInfo) {
+  //...
+});
+```
+
+### Manage events
 
 #### Retrieve
 
 ```javascript
-var filter = new pryv.Filter({limit : 20});
-connection.events.get(filter, function (err, events) {
+var filter = new pryv.Filter({limit : 10});
+connection.events.get(filter, function (err, eventList) {
   // ...
 });
 ```
 
 #### Create
 
 ```javascript
+// This is the minimum required data to create an event
 var eventData = {
-  streamId: 'diary',
+  streamId: 'valid-stream-id',
   type: 'note/txt',
-  content: 'I track, therefore I am.'
+  content: 'This is an example.'
+};
+connection.events.create(event, function (err, eventCreated) { 
+  // ...
+});
+```
+See also: [More about types](/event-types/)
+
+#### Update
+
+```javascript
+event.content = 'This is an update.';
+connection.events.update(event, function (err, eventUpdated) {
+  // ...
+});
+```
+
+#### Delete
+
+```javascript
+// Only the id of an event is needed for removal
```

Comment:
I think you can omit this comment, it's closer to implementation details. So out of scope.